src/System.IO.Packaging/src/project.json

```diff
@@ -0,0 +1,14 @@
+{
+  "dependencies": {
+    "System.Collections": "4.0.10-beta-23008",
```

Comment:
Right now the latest packages aren't compatible with the version of DNX we are using, so if you use the * syntax you will get an incompatible package.